client/dfclient/src/gameplay_state.cpp

```diff
@@ -63,13 +76,22 @@ std::unique_ptr<ncine::AnimatedSprite> GameplayState::CreateNewAnimSprite(ncine:
 		ret->addAnimation(nctl::move(animation));
 	}
 
-	ret->setAnimationIndex(FISH_ANIMATIONS::WALK);
+	ret->setAnimationIndex(0);
 	ret->setFrame(0);
 	ret->setPaused(false);
-	ret->setLayer(MOBS_LAYER);
+	ret->setLayer((unsigned short)Layers::MOBS);
```

Comment:
I think that this should be passed through a parameter.

Reply:
And now it is.